documentation/SolutionsRegistryFormat.md

```diff
@@ -49,6 +49,12 @@ The `settingsHandlers` block is unique and one of the most important blocks in t
         "options": {
             "filename": "${{environment}.APPDATA}\\Texthelp Systems\\RWSettings10.xml"
         },
+        "supportedSettings": : {
+            "appsetting1": { ... appsetting1 metadata ... },
```

Comment:
Is it the string which appears after the mysterious URL prefix "http://registry\\.gpii\\.net/common" ?